tests/cli/commands/test_group.py

```diff
@@ -82,9 +79,9 @@ def test_all(self):
         q = CompsQuery(self.comps, self.prst, kinds_all, status_all)
 
         res = q.get('sugar*', '*er*')
-        self.assertCountEqual(names(res.environments),
```

Comment:
Preserve the old tests with `# compat: erase in 2.0.0` and add to them new ones, please.